cmd/samples/pso/workflow.go

```diff
@@ -89,11 +89,11 @@ func PSOWorkflow(ctx workflow.Context, functionName string) (err error) {
 		}
 
 		var goalReached bool
-		childFuture := workflow.ExecuteChildWorkflow(ctx, PSOChildWorkflow, *swarm, 1)
-		//var childWE WorkflowExecution
-		//childFuture.GetChildWorkflowExecution().Get(&childWE)
-		err = childFuture.Get(ctx, &goalReached)
-		//ctx = workflow.WithValue(ctx, QueryResultName, childWE.RunID)
+		childWorkflowFuture := workflow.ExecuteChildWorkflow(ctx, PSOChildWorkflow, *swarm, 1)
```

Comment:
It looks like the childWorklfowID is the same for each iteration. I would generate a different one for each for clarity and troubleshooting.

Reply:
Right, I'll leave Cadence to generate it for me then